docs/getting-started-guides/gce.md

```diff
@@ -39,20 +39,32 @@ If you want to use custom binaries or pure open source Kubernetes, please contin
 
 ### Starting a cluster
 
-You can install a client and start a cluster with this command:
+You can install a client and start a cluster with either one of these commands (we list both in case only one is installed on your machine):
+
+
+ ```bash
+ curl -sS https://get.k8s.io | bash
+ ```
+
+or
 
 ```bash
-curl -sS https://get.k8s.io | bash
+wget -q -O - https://get.k8s.io | bash
 ```
 
-Once this command completes, you will have a master VM and four worker VMs, running as a Kubernetes cluster. By default, some containers will already be running on your cluster. Containers like `kibana` and `elasticsearch` provide [logging](../logging.md), while `heapster` provides [monitoring](../../cluster/addons/cluster-monitoring/README.md) services.
+Once this command completes, you will have a master VM and four worker VMs, running as a Kubernetes cluster.
+
+By default, some containers will already be running on your cluster. Containers like `kibana` and `elasticsearch` provide [logging](../logging.md), while `heapster` provides [monitoring](../../cluster/addons/cluster-monitoring/README.md) services.
+
+The script run by the commands above creates a cluster with the name/prefix "kubernetes". It defines one specific cluster config, so you can't run it more than once.
```

Comment:
FYI: I think you can do:
```
export KUBE_GCE_INSTANCE_PREFIX="cluster2"
curl -sS https://get.k8s.io | bash
```
and get a second cluster called cluster2.

You don't have to add that to the docs.  Just fyi.